field_rrule/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+================
+Repetition rules
+================
+
+This module was written to offer a field type that holds rrules according to RFC 2445.
```

Comment:
I linked what the author of dateutil claims to have implemented: https://labix.org/python-dateutil#head-470fa22b2db72000d7abe698a5783a46b0731b57